README.md

```diff
@@ -1 +1,179 @@
+
+This site is built as part of an exercise at Microverse. The site is about a SpaceX, This project is created with react js👌 and Redux as a state managment library.
+
+**The Currency Metric App React project** i
+
+## 🛠 Built With <a name="built-with">React</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML & CSS</summary>
+  <ul>
+    <li><a href="#">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a href="#">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display world currencies**
+- **Filter by currency**
+- **Display the exchange rate of currencies**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://currency-exchange-app.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to follow the below steps
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/abeltsew/spacex.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  node.js V16 and above
+```
+
+### Usage
+
+To run the project:
+
+```sh
+  use git bash to open in Vs code
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx hint .
+
+  npx stylelint "**/*.{css,scss}" '
+```
+
+to check linter errors.
+
+## 👥 Authors <a name="authors"></a>
+
+\*\*Authors:
+
+👤 **Prince**
+
+- GitHub: [@prius2055](https://github.com/prius2055)
+- Twitter: [@prius2055](https://www.twitter.com/prius2055)
+- LinkedIn: [@prius2055](https://www.linkedin.com/prius2055)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Live Chat**
+- [ ] **More currencies**
+- [ ] **Youtube Integration**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can buy me coffee. Contact detail is above.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for their support.
+
+I would also like to appreciate the creative commons team for the graphical design and particularly Nelson Sakwa the author of the design.
```

Comment:
- [ ] Please note that the creative commons is an organization that provides free licences for creators to use. It is not someone who provides you designs. The actual designer is [Nelson Sakwa](https://www.behance.net/sakwadesignstudio). Kindly make sure to give appropriate credits to the designer as suggested earlier ⛏🤓, and you can mention creative commons like the following. Please feel free to use it 👇

  I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Fixed